Add description of vshard.*.info() options (#3306)
tarantool/doc: doc/reference/reference_rock/vshard/vshard_storage.rst

..  _storage_api-info:

..  function:: vshard.storage.info({options})

>     Return information about the storage instance. Since vshard v.0.1.22 the

Reviewer: ```suggestion
    Return information about the storage instance. Since vshard v.0.1.22, the
```

Author: Applied

    function also accepts options, which can be used to get additional
    information.

    :param options:

       *    ``with_services`` — a bool value. If set to ``true``, the

>             function returns information about background services

Reviewer: ```suggestion
            function returns information about the background services
```

            (such as gc, rebalancer, recovery or applier of the routes),

>             which are working on the current instance. See

Reviewer: ```suggestion
            function returns information about the background services
            (such as garbage collector, rebalancer, recovery, or applier of the routes)
            that are working on the current instance. See
```

Author: Applied
